src/s3.js

```diff
@@ -57,7 +52,7 @@ module.exports = function s3UploadFactory(knoxOpt, opt) {
 		const params = {
 			ACL: 'public-read',
 			Body: file.contents,
-			Bucket: knoxOpt.bucket,
+			Bucket: 'd2lprodcdn',
```

Comment:
Are we good with this not being configurable?

Reply:
I think it was always hard coded https://github.com/Brightspace/frau-publisher/blob/master/src/optionsValidator.js#L69